packages/server/migrations/20240330125917_make_teamName_teamCode_unique.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+exports.up = function (knex) {
+  return knex.schema.alterTable('Teams', async () => {
+    await knex.raw(`ALTER TABLE Teams ADD UNIQUE (team_name);`);
+    await knex.raw(`ALTER TABLE Teams ADD UNIQUE (team_code);`);
+  });
+};
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.down = function () {};
```

Comment:
Will be suggested to add a down function as well

```suggestion
exports.down = async knex => {
  await knex.schema.alterTable('lenusBenefit', table => {
     table.string('team_name').alter()
     table.string('team_code').alter()
  })
}

```